docs/sources/reference/components/loki/loki.secretfilter.md

```diff
@@ -0,0 +1,117 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/loki/loki.secretfilter/
+description: Learn about loki.secretfilter
+title: loki.secretfilter
+labels:
+  stage: experimental
+---
+
+<span class="badge docs-labels__stage docs-labels__item">Experimental</span>
+
+# loki.secretfilter
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`loki.secretfilter` receives log entries and redacts sensitive information, such as secrets, from them.
+
+## Usage
+
+```alloy
+loki.secretfilter "LABEL" {
+    forward_to = RECEIVER_LIST
```

Comment:
```suggestion
loki.secretfilter "<LABEL>" {
    forward_to = <RECEIVER_LIST>
```

Use `<` and `>` around replaceable elements this for (future) consistency. The docs are being aligned to use this syntax.

Reply:
Changed in 500891fa01cbc83dac17ae4ea6fce500f2a1d8fb.